Dockerfile

```diff
@@ -10,6 +10,9 @@ RUN wget --no-check-certificate https://dl.xpdfreader.com/xpdf-tools-linux-4.03.
 
 RUN apt-get install libpoppler-cpp-dev pkg-config -y --fix-missing
 
+# Install Tesseract
+RUN apt-get install tesseract-ocr libtesseract-dev poppler-utils
+
```

Comment:
If your intention is to make tesseract available in the CI, this will be the right place to install tesseract https://github.com/deepset-ai/haystack/blob/master/.github/workflows/ci.yml#L85

The dockerfile here is the public one on dockerhub (e.g. used in the docker-compose demo, deployments etc.)

Reply:
I believe my intention was to add to both places, I have added the dependency sizes on the other comment, will make changes or keep it as it is, based on your response there.